### DISCUSSION TITLE
参考文献部分网站引用无法换行

### DISCUSSION THREAD

**Discussion by LILKOTYO** [sjtug/SJTUThesis#633] [提问] [answered]
我在文章中引用了一个网站，其网站地址较长，例如下面这个样子，引文内容无法自动换行导致超过纸张宽度，请问这种情况该如何解决呢？

某网站[Z]. https://abababbabababababbabababababababbaba/index.html.

**Comment by skyzh** [comment]
<img width="914" alt="image" src="https://user-images.githubusercontent.com/4198311/119294060-34a80c00-bc86-11eb-8f74-37f7a29ffcc4.png">

我这边是可以正常换行的，可以提供一个具体的 bib entry 吗？

**Reply by LILKOTYO** [reply]
您好，我这边自己偶然解决了，之前我是用zotero来进行文献的管理，导出BibTeX格式后的内容是这样的:
@misc{MoKuai,
  title = {模块},
  abstract = {UBT模块概述，包括属性的介绍。},
  howpublished = {https://abababbabababababbabababababababbaba/index.html},
  language = {en-US}
}
后来我参考其他网站应用的内容将howpublished更改为了url，虽然我不太清楚原因，但这样操作后就成功正常显示了。

**Comment by AlexaraWu** [comment, marked as answer]
模板有对 url 的断行作特殊处理，请使用 url 域或者 `\url` 命令。

```bib
@misc{MoKuai,
  title        = {模块},
  abstract     = {UBT模块概述，包括属性的介绍。},
  howpublished = {\url{https://abababbabababababbabababababababbaba/index.html}},
  language     = {en-US}
}
```
